config.yaml.sample

```diff
@@ -50,6 +50,12 @@ all:
 
   # Should newly discovered nodes be marked installed?
   protect_new_nodes: false
+  
+  # Invert the 'protect_new_nodes' value for the following subnets
+  # List subnets in the CIDR format (a.a.a.a/bb)
+  #
+  # invert_protect_new_nodes_for_subnets:
```

Comment:
I'm wary of a property that inverts the value of another property. Toggling `protect_new_nodes` completely changes the behavior of this property. One could feasibly toggle the above value without reading this comment. Did you consider a less contingent property or properties, maybe `protect_new_nodes_on_subnet` and/or `provision_new_nodes_on_subnet`?